documentation/data-importer.adoc

```diff
@@ -138,6 +136,28 @@ If you're unsure if you've followed the steps correctly so far, you can go ahead
 button::Load Northwind node mapping[role=NX_IMPORT_LOAD,endpoint={northwind-subset-node-only-mapping-zip-file}]
 ====
 
+== Constraints and indexes
+
+To ensure accuracy of your data and support read performance, Data Importer automatically creates a uniqueness constraint and a corresponding index on the property you select as the node ID.
+You can't modify the two that are created for you, but you can add additional indexes.
```

Comment:
This feels negative. Could you spin this as a positive e.g.

In addition to the mandatory constraint and indexes and you can add additional indexes to improve query performance.

Reply:
You're right! Done!